src/main/java/hlf/java/rest/client/service/impl/ChannelServiceImpl.java

```diff
@@ -116,6 +129,17 @@ public ClientResponseModel joinChannel(ChannelOperationRequest channelOperationR
     }
   }
 
+  @Override
+  public Set<String> getChannelMembersMSPID(String channelName) {
+
+    if (StringUtils.isEmpty(channelName)) {
```

Comment:
[nitpick] you can prefer StringUtils#isBlank over isEmpty.
The latter return false for string containing only whitespace `" "`

Reply:
changes done